src/scripts/scrumHelper.js

```diff
@@ -113,22 +115,26 @@ function allIncluded() {
 			'-' +
 			('00' + lastWeekMonth.toString()).slice(-2) +
 			'-' +
-			('00' + lastWeekDay.toString()).slice(-2);
+			('00' + lastWeekDay.toString()).slice(-2) +
+			'T00:00:00Z';
 		return lastWeekDisplayPadded;
 	}
 	function getToday() {
 		var today = new Date();
-		var Week = new Date(today.getFullYear(), today.getMonth(), today.getDate());
-		var WeekMonth = Week.getMonth() + 1;
-		var WeekDay = Week.getDate();
-		var WeekYear = Week.getFullYear();
-		var WeekDisplayPadded =
-			('0000' + WeekYear.toString()).slice(-4) +
+		today.setHours(23, 59, 59, 999);
+		var week = new Date(today.getFullYear(), today.getMonth(), today.getDate());
```

Comment:
Your variable name is confusing. Its name is `week`, but the value seems to be a single date.